### ISSUE TITLE
Can't run katago 1.14 on Ubuntu 22.04 missing libzip.so.5

### ISSUE THREAD

**Issue by emogenet** [lightvector/KataGo#875]
Downloaded katago-v1.14.0-eigenavx2-linux-x64+bs29.zip

Extracted in a folder

Tried to run it via:

    ./katago

Error message:

    ./katago: error while loading shared libraries: libzip.so.5: cannot open shared object file: No such file or directory

Turns out libzip5 is a nightmare to install on ubuntu 22.04 because of a dependency if libzip5 on an antique version of libssl that can't easily be installed on 22.04 either.

Do we need to depend on OpenSSL just to be able to play go ... strikes me as overkill.

**Comment by lightvector** [comment]
Yes dependencies can be a bit messy. As described in the release notes, if you have this issue, try compiling KataGo yourself - clone the git repo and follow the instructions at https://github.com/lightvector/KataGo/blob/master/Compiling.md.

KataGo is easy to compile on ubuntu - every dependency you need is also installable via apt.

**Comment by akdtg** [comment]
From https://askubuntu.com/questions/1437133/does-ubuntu-22-04-support-the-libzip5-package:

You can install it from the https://packages.ubuntu.com/focal/amd64/libzip5/download.

```
wget http://mirrors.kernel.org/ubuntu/pool/universe/libz/libzip/libzip5_1.5.1-0ubuntu1_amd64.deb
sudo apt install ./libzip5_1.5.1-0ubuntu1_amd64.deb
```

**Comment by jackiesteed** [comment]
It also happens on aliyun centos server. why is the libzip version is 5, not a higher version?

**Comment by jackiesteed** [comment]
https://lifein19x19.com/viewtopic.php?f=11&t=19050
many people have problems about the dependencies, especially libzip. @lightvector 
<img width="672" alt="image" src="https://github.com/user-attachments/assets/141b9a61-bc1a-48b6-b13a-aa1fe6b53ca9">

**Comment by siasio** [comment]
> From https://askubuntu.com/questions/1437133/does-ubuntu-22-04-support-the-libzip5-package:
> 
> You can install it from the https://packages.ubuntu.com/focal/amd64/libzip5/download.
> 
> ```
> wget http://mirrors.kernel.org/ubuntu/pool/universe/libz/libzip/libzip5_1.5.1-0ubuntu1_amd64.deb
> sudo apt install ./libzip5_1.5.1-0ubuntu1_amd64.deb
> ```

Before running the installation you suggested, I need to run (as suggested [here](https://askubuntu.com/a/1403683)):

```
echo "deb http://security.ubuntu.com/ubuntu focal-security main" | sudo tee /etc/apt/sources.list.d/focal-security.list

sudo apt-get update
sudo apt-get install libssl1.1
```
I'm posting it in this thread so that next time I don't need to google the same error and look for this snippet again :D
